src/main/java/com/example/java/maven/gameBlackJack/Game.java

```diff
@@ -0,0 +1,96 @@
+package com.example.java.maven.gameBlackJack;
+
+public class Game {
+    Player player = new Player(UserInputProvider.getPlayersName());
```

Comment:
move to constructor, should be private final :P

Reply:
IMHO, also Player and Croupier creation should be done outside of the Game and these object should be passed to game via constructor parameter (dependency injection)

https://pl.wikipedia.org/wiki/Wstrzykiwanie_zależności
https://typeofweb.com/wzorce-projektowe-dependency-injection/

Reply:
Done